source/wp-content/themes/wporg-developer/inc/search.php

```diff
@@ -85,11 +94,11 @@ public static function pre_get_posts( $query ) {
 		} else {
 			// If user has '()' at end of a search string, assume they want a specific function/method.
 			$s = htmlentities( $query->get( 's' ) );
-			if ( '()' === substr( $s, -2 ) ) {
+			if ( '()' === substr( $s, -2 ) || '(' == substr( $s, -1 ) ) {
```

Comment:
This was to ensure that a search like `wp_authenticat(` would return results, rather than that single char having no results.

`parse_query` is a good test phrase too.